frontend/ui/widget/doublespinwidget.lua

```diff
@@ -177,7 +177,7 @@ function DoubleSpinWidget:update(numberpicker_left_value, numberpicker_right_val
                     right_widget.value = self.right_default
                     left_widget:update()
                     right_widget:update()
-                    self.callback(nil, nil)
+                    self.callback(self.left_default, self.right_default)
```

Comment:
Please note this change breaks
https://github.com/koreader/koreader/blob/4e517f4839182ab78d199a3bc2beab6fc3ecd297/frontend/apps/reader/modules/readertypography.lua#L386

Reply:
@hius07 Thanks for your hint, but I don't see why.

My addition is just in the arguments of the callback for default_callback. It is stated "Apply..."
https://github.com/koreader/koreader/blob/4e517f4839182ab78d199a3bc2beab6fc3ecd297/frontend/ui/widget/doublespinwidget.lua#L172

and this modification does apply the values.

Ohhhh, I see now, the `Apply` button is enabled with my change, but the values are already applied.

Really nice catch :100: 

Do you think, we should change the button text from "Apply ..." to "Set ..." or "Flipp ..." or "Turn ..." or "Change ..." or even better "Spin to default values"? This would make clear the button does not apply the values directly.

Reply:
These `(nil, nil)` were especially introduced for readertypography by @poire-z.
Default values of the hyphenation limits are not stored in the settings and are applied when the settings are nil (absent).